### DISCUSSION TITLE
XRSI + OMI Discussion

### DISCUSSION THREAD

**Discussion by robertlong** [omigroup/omigroup#39] [General]
@CyberGuardian-XRSI (Kavya Pearlman) brought up some possibilities for XRSI to collaborate with OMI and for OMI to integrate their diversity and inclusion goals. We'll be meeting on 5/28/2021 at 12:00 PM PDT in the AngellXR Discord to discuss this topic more. It's an open forum, please feel free to attend, ask questions and participate in these discussions.

**Comment by mrmetaverse** [comment]
Bump to continue this conversation.

**Reply by CyberGuardian-XRSI** [reply]
Hia, Happy to hop into discord discussion anytime for this.. tell me when , my discord is Kavya Pearlman#1640
